content/news/050/index.md

```diff
@@ -246,14 +243,14 @@ _Discussions: [GitHub][jumpy_discussions], [Twitter][jumpy_twitter]_
 
 ## Engine Updates
 
-### Bottomless-Pit 0.3.1
```

Comment:
I think it's worth leaving in version number, since it suggests a new version was released, instead of just general progress on a project. For example, I imagine we'd have a Bevy release's section titled `Bevy 0.XX` to signify the importance.

Reply:
Good point, but then I'd like to omit the patch version, as otherwise it seems like we will be talking about some minor bugfixes only. As far as I can tell, this entry concerns `0.13.0` and `0.13.1`, so I'll write it as `0.13`

Reply:
[`44c261b` (#1496)](https://github.com/rust-gamedev/rust-gamedev.github.io/pull/1496/commits/44c261b5cd65f664854a03a9ead0951a1d145f9a)